src/pages/sitemap/Sitemap.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import Typography from '@material-ui/core/Typography';
+import Link from '@material-ui/core/Link';
+import { useTranslation } from 'react-i18next';
+import styled from 'styled-components';
+import { PageHeader } from '../../components/PageHeader';
+import { StyledContentWrapper } from '../../components/styled/Wrappers';
+import { API_PATHS, API_URL } from '../../utils/constants';
+import { logout } from '../../api/userApi';
+import { List, ListItem } from '@material-ui/core';
+
+const StyledTypography = styled(Typography)`
+  margin-top: 2rem;
+`;
+
+interface LinkAndDescription {
+  href: string;
+  description: string;
+  onClickFunction?: () => void;
+}
+
+const Sitemap = () => {
+  const { t } = useTranslation();
+  const currentUrl = encodeURIComponent(
+    `${window.location.protocol}//${window.location.hostname}${window.location.port ? ':' + window.location.port : ''}`
+  );
+
+  const handleLogout = () => {
+    logout().then(() => {
+      localStorage.removeItem('token');
```

Comment:
her burde vi vel heller brukt en felles logout-funksjon - så slipper vi å måtte vedlikeholde kode på to steder.